docs/source/cli/model-training-inference/real-time-inference-spec.rst

```diff
@@ -0,0 +1,229 @@
+  depending on the value of ``gml_task``.
+
+Contents of a ``node`` object listed in a ``nodes`` array
+..........................................................
+
+A ``node`` object listed in a ``nodes`` array can contain the following required fields.
+
+.. code:: json
+
+    {
+        "node_type" : string,
+        "node_id"   : string,
+        "features"  : {
+                        feature name: feature value,
+                        feature name: feature value,
+                        ...
+                      }
+    }
+
+* ``node_type`` -- A string specifying the node type name in a graph. It should be same as these
+  ``node_type`` defined in :ref:`gconstruct JSON specification <gconstruction-json>` or the ``type``
+  values of ``nodes`` defined in :ref:`gsprocessing JSON specification <gsprocessing_input_configuration>`.
+* ``node_id`` -- A string specifying the node identifier.
+* ``features`` -- A dictionary, whose key is a feature name, and its value is the value of features.
+  feaure names should be same as the ``feature_name`` defined in :ref:`gconstruct JSON specification
+  <gconstruction-json>`, or these ``name`` values of ``features`` fields defined in
+  :ref:`gsprocessing JSON specification <gsprocessing_input_configuration>`.
+
+Contents of an ``edge`` object listed in an ``edges`` array
+............................................................
+
+An ``edge`` object listed in an ``edges`` array can contain the following required fields.
+
+.. code:: json
+
+    {
+        "edge_type"     : [(source node type), (edge type), (destination node type)],
+        "src_node_id"   : string,
+        "dest_node_id"  : string,
+            "features"  : {
+                            feature name: feature value,
+                            feature name: feature value,
+                            ...
+                        }
+    }
+
+* ``edge_type`` -- An array specifying the edge type name in the format of three strings, which indicate the
+  source node type, the edge type, and the destination edge type. It should be same as the ``relation`` fileds
+  defined in :ref:`gconstruct JSON specification <gconstruction-json>` or the ``type`` values of ``source``
+  ``relation``, and ``dest`` fileds defined in
+  :ref:`gsprocessing JSON specification <gsprocessing_input_configuration>`.
+* ``src_node_id`` -- A string specifying the source node identifier.
+* ``dest_node_id`` -- A string specifying the destination node identifier.
+* ``features`` -- A dictionary, whose key is a feature name, and its key is value of the feature. 
+  feaure names should be same as these ``feature_name`` defined in :ref:`gconstruct JSON specification
+  <gconstruction-json>`, or these ``name`` values of ``features`` fields defined in
+  :ref:`gsprocessing JSON specification <gsprocessing_input_configuration>`.
+
+Contents of a target object listed in a ``targets`` array
```

Comment:
```suggestion
Contents of a `target` object listed in a ``targets`` array
```

Reply:
There is no `target` object. It is either a ``node`` or ``edge``. So here use target purposely.